pl_bolts/models/gans/srgan/components.py

```diff
@@ -0,0 +1,150 @@
+"""
+based on
+https://colab.research.google.com/github/https-deeplearning-ai/GANs-Public/blob/master/C3W2_SRGAN_(Optional).ipynb
+"""
+import torch
+import torch.nn as nn
+
+from pl_bolts.utils import _TORCHVISION_AVAILABLE
+from pl_bolts.utils.warnings import warn_missing_pkg
+
+if _TORCHVISION_AVAILABLE:
+    from torchvision.models import vgg19
+else:
+    warn_missing_pkg("torchvision")  # pragma: no-cover
+
+
+class ResidualBlock(nn.Module):
```

Comment:
I think Residual Block is far too often used in our codebase (SSL too) 
Marking it for #427

Reply:
@oke-aditya Thanks for marking it :]